tests/gdb-tests/tests/test_command_stepuntilasm.py

```diff
@@ -0,0 +1,29 @@
+import gdb
+import pytest
+
+import pwndbg.gdblib
+import tests
+
+STEPUNTILASM_X64_BINARY = tests.binaries.get("stepuntilasm_x64.out")
+
+
+@pytest.mark.parametrize("binary", [STEPUNTILASM_X64_BINARY], ids=["x86-64"])
```

Comment:
Do we need to parametrize this?

Reply:
(I get the idea that it would be great to do so, but for now we only have x64 here)

Reply:
https://docs.pytest.org/en/7.3.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions

The `parametrize` decorator allows you to provide testcases for the given test function so that each testcase will be executed as a separate test.

For now, we execute all tests on x86-64 only and whenever there is a need for a different architecture I believe it is handled separately, usually by running it via QEMU-user emulation and connecting to its gdbstub/gdbserver server.

Reply:
So we can just remove that parametrize here

Reply:
Oh, perfect, then. I'll remove the parametrization